### PR TITLE
Frontend refactor banner

### PR DESCRIPTION
**Pull request by thisconnect**
This probably could use useSync hook, but keeping it simple first.

The banner is shown but there is a subscription which I am not sure how to test.

### CODE REVIEW

File: frontends/web/src/components/banner/banner.tsx [hunk 1]
```diff
@@ -14,53 +15,59 @@
...
       )}
     </Status>
   );
-}
-
-const HOC = translate()(
-  subscribe<TLoadedProps, TBannerProps & TranslateProps>(
-    ({ msgKey }) => ({ banner: 'banners/' + msgKey }),
-    true,
-    false,
```
Comment (thisconnect) [inline frontends/web/src/components/banner/banner.tsx:62]: the true false here means renderOnlyOnceLoaded and subscribeWithoutLoading .

**Review by shonsirsha — APPROVED** [review]
untested, code LGTM.

**Comment by thisconnect** [comment]
rebased

**Comment by thisconnect** [comment]
move backend api functions to `api/banners`

**Comment by shonsirsha** [comment]
>  move backend api functions to api/banners



even better :)

**Comment by thisconnect** [comment]
rebased and fixed merge conflict.

I still want to do some more testing.

**Comment by thisconnect** [comment]
rebased

**Comment by thisconnect** [comment]
rebased

**Pull request merged**